.github/ISSUE_TEMPLATE/ios_background_message_handling.yml

```diff
@@ -0,0 +1,170 @@
+
+        <details>
+        <summary>Click to see how your Xcode project "Signing and Capabilities" settings should look like</summary>
+        <img alt="Xcode "Signing and Capabilities tab" src="./../images/remote-notifications-enable.png?raw=true" />
+        </details>
+
+        REQUIRED: Please provide a screenshot of your Xcode project "Signing and capabilities" settings in the text area.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Have you created an APNs key in your Apple Developer account & uploaded this APNs key to your Firebase console?
+      description: |
+        You have to create an APNs key in your [Apple Developer account](https://developer.apple.com/membercenter/index.action) & upload this APNs key to your [Firebase console](https://console.firebase.google.com). [See documentation](https://firebase.google.com/docs/cloud-messaging/flutter/client#upload_your_apns_authentication_key).
+
+        <details>
+        <summary>Click to see how an APNs key uploaded to the correct app in the Firebase console should look like</summary>
+        <img alt="APNs key uploaded to Firebase console" src="./../images/apns-key-upload.png?raw=true" />
+        </details>
+
+        REQUIRED: Please provide a screenshot of your APNs key uploaded to your Firebase console to the correct app in the text area.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Have you enabled method swizzling in your app?
+      description: |
+        Go to your "Info.plist" file and add the following key `FirebaseAppDelegateProxyEnabled` with a value of `YES`.
+
+        REQUIRED: Please paste your Info.plist in the text area.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Are you sending messages to your app from the Firebase Admin SDK?
+      description: |
+        For an example of sending messages to your app, please see [FlutterFire's nodejs Firebase Admin SDK script](https://github.com/firebase/flutterfire/blob/master/packages/firebase_messaging/firebase_messaging/example/scripts/send-message.js). FlutterFire does not support sending messages from 3rd party packages.
+
+        It is essential you include the `contentAvailable` flag in your message payload when sending messages to your app from the Firebase Admin SDK.
+
+        Example code snippet using the nodejs Firebase Admin SDK to send a message to your app:
+        ```js
+        admin
+          .messaging()
+          .sendToDevice(
+            [token],
+            {
+              data: {
+                foo:'bar',
+              },
+              notification: {
+                title: 'A great title',
+                body: 'Great content',
+              },
+            },
+            {
+              // Required for background/terminated app state messages on iOS
+              contentAvailable: true,
+            }
+          )
+        ```
```

Comment:
```suggestion
	<details>
	<summary>View example code snippet for using the Node.js Firebase Admin SDK to send a message to your app</summary>
        ```js
        admin
          .messaging()
          .sendToDevice(
            [token],
            {
              data: {
                foo:'bar',
              },
              notification: {
                title: 'A great title',
                body: 'Great content',
              },
            },
            {
              // Required for background/terminated app state messages on iOS
              contentAvailable: true,
            }
          )
        ```
        </details>
```